File: sync/src/main/scala/com/fan/mysql/event/impl/XidEvent.java
package com.dingcloud.dts.binlog.event.impl;

import com.dingcloud.dts.binlog.event.AbstractEvent;
import com.dingcloud.dts.binlog.event.EventHeader;

public class XidEvent extends AbstractEvent {

	private long xid;

	public XidEvent(EventHeader header) {
		super(header);
	}

	public long getXid() {
		return xid;
	}

	public void setXid(long xid) {
		this.xid = xid;
	}

}

File: async/src/main/scala/com/fan/mysql/async/binlog/event/impl/XidEvent.scala
package com.fan.mysql.async.binlog.event.impl

import com.fan.mysql.async.binlog.event.{AbstractEvent, EventHeader}

class XidEvent(header: EventHeader) extends AbstractEvent(header) {

  private[this] var xid: Long = _

  def getXid: Long = xid

  def setXid(xid: Long): Unit = {
    this.xid = xid
  }
}
